staubli_tx90_gazebo/launch/tx90_gazebo.launch

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="paused" default="false"/>
+  <arg name="gui" default="true"/>
+  <arg name="remap" default="false"/>
```

Comment:
Sounds good!
Shall we do that for the RX160 family too?
I'll address this soon-ish.